packages/components/src/components/Snackbar/SnackbarProvider.tsx

```diff
@@ -12,7 +12,7 @@ type SnackbarContextType = {
 }
 
 export const SNACKBAR_DURATIONS = {
-  DEFAULT: 1000000000000, // Essentially indefinite until dismissed
+  DEFAULT: 2147483646, // Essentially indefinite until dismissed (23 days, limited by 32-bit int)
```

Comment:
Not an issue with the code itself, but the unit tests run in node which only allows 32-bit integers which go from -2147483647 to 2147483647 and the overflow was setting it to 1 second. Adjusted down to make the unit tests work and 23 days and change is still plenty to be "indefinite" for practical purposes.